core/common/src/main/java/alluxio/util/JvmPauseMonitor.java

```diff
@@ -90,11 +88,10 @@ public void stop() {
 
   /**
    * Resets value of mJvmMonitorThread and mThreadStarted.
-   * @return reseted mThreadStarted
+   * @return reseted JvmPauseMonitor
```

Comment:
`mJvmMonitorThread `

Reply:
I will change the return value of this function to `void`, is that ok?